academy/3-my-own-chain/ignitecli.md

```diff
@@ -122,59 +128,162 @@ Available Commands:
+</CodeGroupItem>
+
+</CodeGroup>
+
+`github.com/alice/checkers` is the name of the Golang module by which this project will be known. And if you own the `github.com/alice` path, you can even eventually host it there and have other people use your project as a module.
+
+<ExpansionPanel title="Troubleshooting">
 
-<p>&nbsp;</p>
+For the sake of good support, the versions of all software used are communicated as encountered throughout this course. It is natural that after the writing of the course material some version changes will appear, and it may occur that something breaks. Instead of using different versions of the software from the ones in the course, please look at the following list, which might fix problems you are running into. Otherwise, use Docker, as explained on this page.
+<p></p>
+If all else fails, please post the issue you face on Discord.
 
 <PanelListItem number="1">
 
 **Apple M1**
 
-If you work with a machine using M1 architecture, make sure:
+If you work with a machine using M1 architecture, the Docker image should allow you to run with your specific CPU architecture. However, if you encounter too many problems, you can try the following:
 
-1. You follow this course in a [Rosetta](https://www.courier.com/blog/tips-and-tricks-to-setup-your-apple-m1-for-development/) terminal. 
+1. You follow this course in a [Rosetta](https://www.courier.com/blog/tips-and-tricks-to-setup-your-apple-m1-for-development/) terminal.
 2. You install [Homebrew](https://brew.sh/index).
 3. You install Golang with `brew install go`.
 
 </PanelListItem>
 
-<PanelListItem number="2">
+<PanelListItem number="2" :last="true">
 
 **Building Errors during `scaffold`**
 
 If you work with Go 1.18, you may need to install the following:
 
-* go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@latest
-* go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@latest
-* go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@latest
-* git clone https://github.com/regen-network/cosmos-proto
-  cd cosmos-proto/protoc-gen-gocosmos
-  go install
-* go get github.com/golangci/golangci-lint/cmd/golangci-lint
-* go get golang.org/x/crypto/ssh/terminal@v0.0.0-20220411220226-7b82a4e95df4
+* ```sh
+  $ go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@latest
+  ```
+
+* ```sh
+  $ go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@latest
+  ```
+
+* ```sh
+  $ go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@latest
+  ```
+
+* ```sh
+  $ git clone https://github.com/regen-network/cosmos-proto
```

Comment:
To be modified with github.com/cosmos/gogoproto's `protoc-gen-gocosmos` eventually

Reply:
Are you saying that Ignite is changing the versions it uses?